### PR TITLE
Security docs update for v8

### PR DESCRIPTION
**Pull request by elit0451**
Updating security articles so that they are valid for Umbraco 8.

### CODE REVIEW

**Comment by sofietoft** [comment]
Looking great so far @elit0451 ! 😄 👏

File: Extending/Health-Check/index.md [hunk 1]
```diff
@@ -1,37 +1,35 @@
 ---
-versionFrom: 7.0.0
-needsV8Update: "true"
+versionFrom: 8.0.0
 ---
 
 # Health Check
 
-The developer section of the Umbraco backoffice holds a dashboard named "Health Check". It is a handy list of checks to see if your Umbraco installation is configured according to best practices. It's possible to add your custom built health checks.
+The settings section of the Umbraco backoffice holds a dashboard named "Health Check". It is a handy list of checks to see if your Umbraco installation is configured according to best practices. It's possible to add your custom built health checks.
```
Comment (sofietoft) [inline Extending/Health-Check/index.md]:
```suggestion
The Settings section of the Umbraco backoffice holds a dashboard named "Health Check". It is a handy list of checks to see if your Umbraco installation is configured according to best practices. It's possible to add your custom built health checks.
```

File: Extending/Health-Check/index.md [hunk 2]
```diff
@@ -1,37 +1,35 @@
...
+  * **Content/MIME Sniffing Protection (id: `1CF27DB3-EFC0-41D7-A1BB-EA912064E071`)** - checks that your site contains a header used to protect against MIME sniffing vulnerabilities
+  * **Cookie hijacking and protocol downgrade attacks Protection (HSTS) (id: `E2048C48-21C5-4BE1-A80B-8062162DF124`)** - checks if your HTTPS site contains the Strict-Transport-Security Header (HSTS). If not - adds with a default of 18 weeks
+  * **Cross-site scripting Protection (id: `F4D2B02E-28C5-4999-8463-05759FA15C3A`)** - checks for the presence of the X-XSS-Protection-header
   * **Excessive Headers (id: `92ABBAA2-0586-4089-8AE2-9A843439D577`)** - checks to ensure that various headers that can provide details about the technology used to build and host the website have been removed
   * **HTTPS check (id: `EB66BB3B-1BCD-4314-9531-9DA2C1D6D9A7`)** - to determine if the current site is running on a secure connection
-  * **UmbracoUseSSL check** - when the site is running on HTTPS, `umbracoUseSSL` needs to be enabled to secure the backoffice
```
Comment (sofietoft) [inline Extending/Health-Check/index.md]: Maybe I'm wrong here, but doesn't it still check on the HTTPS setting in web.config file? 
The setting has changed though: https://our.umbraco.com/documentation/Reference/Config/webconfig/#umbracocoreusehttps - but pretty sure it's still included in the health check?

Comment (elit0451) [reply Extending/Health-Check/index.md]: You are right! We still have that check 😁 Nice spot!

**Pull request merged**